cmd/reconcile.go

```diff
@@ -0,0 +1,208 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
```

Comment:
I was sure I left a comment about this but looks like I forgot it 😅 We shouldn't use the `log` stdlib, in all other projects we use [`logrus`](https://github.com/sirupsen/logrus) and we should use it for this project too.